Source/AlamofireExtended.swift

```diff
@@ -23,13 +23,19 @@
 //
 
 /// Type that acts as a generic extension point for all `AlamofireExtended` types.
+@dynamicMemberLookup
 public struct AlamofireExtension<ExtendedType> {
-    /// Stores the type or metatype of any extended type.
-    let type: ExtendedType
+    /// Stores the type or meta-type of any extended type.
+    public private(set) var type: ExtendedType
 
-    init(_ type: ExtendedType) {
+    public init(_ type: ExtendedType) {
         self.type = type
     }
+
+    public subscript<Property>(dynamicMember keyPath: WritableKeyPath<ExtendedType, Property>) -> Property {
```

Comment:
I thought it would inherit one from `@dynamicMemberLookup`. Added in https://github.com/Alamofire/Alamofire/pull/3075/commits/9690297e9af6b3d369e8210732e7b60a722e4c0c.